pkg/controllers/register.go

```diff
@@ -126,23 +126,19 @@ func ExecuteInitializers(controllerCtx *config.ControllerContext, syncers []sync
 	registerContext := util.ToRegisterContext(controllerCtx)
 
 	// execute in parallel because each one might be time-consuming
```

Comment:
This comment should probably be updated too

Reply:
Yep true